pkg/scheduler/framework/interface.go

```diff
@@ -91,6 +91,8 @@ const (
 	// Wait is used when a Permit plugin finds a pod scheduling should wait.
 	Wait
 	// Skip is used when a Bind plugin chooses to skip binding.
+	// Also, if a PreFilter plugin returns skip, it means the plugin's Filter extension point doesn't do anything to that Pod,
```

Comment:
Or more simple, `Also, if a PreFilter plugin returns Skip, coupled Filter plugin will be skipped.`